src/functions/index.ts

```diff
@@ -235,3 +242,82 @@ function hasStringMessage(obj: unknown): obj is { message: string } {
     typeof (obj as { message: string }).message === "string"
   );
 }
+
+function createAutocompleteArgumentsProvider(formulaName: string, args: ArgDefinition[]) {
+  for (let i = 0; i < args.length; i++) {
+    const proposalValues = args[i].proposalValues;
+    if (proposalValues === undefined || proposalValues.length === 0) {
+      continue;
+    }
+
+    const getProposals = (tokenAtCursor: EnrichedToken) => {
+      const functionContext = tokenAtCursor.functionContext;
+      if (
+        !functionContext ||
+        functionContext.parent.toUpperCase() !== formulaName.toUpperCase() ||
+        functionContext.argPosition !== i
+      ) {
+        return;
+      }
+
+      const proposals: AutoCompleteProposal[] = [];
+      let value = "";
+      for (const proposalValue of proposalValues) {
+        switch (typeof proposalValue.value) {
+          case "string":
+            value = `"${proposalValue.value}"`;
+            break;
+          case "number":
+            value = proposalValue.value.toString();
+            break;
+          case "boolean":
+            value = proposalValue.value ? "TRUE" : "FALSE";
+            break;
+          default:
+        }
+
+        proposals.push({
+          text: `${value}`,
+          description: proposalValue.label,
+          htmlContent: [
+            typeof value === "string"
+              ? { value, color: tokenColors.STRING }
+              : { value, color: tokenColors.NUMBER },
```

Comment:
not sure why but booleans are green while they should be blue
<img width="352" height="210" alt="image" src="https://github.com/user-attachments/assets/a87b59da-52c6-4181-8668-ec1cd63dab5b" />